### PR TITLE
Add mysql integration test dabase and env variables

### PR DESCRIPTION
**Pull request by dcabrejas**
Ready for Merge: **Yes**

Adds a new bash script which creates an integration test database when the DB container is first created. This script is mounted as a volume in a special directory of the DB container from which init scripts are loaded and executed on start-up. 

Other than that, it also adds new environment variables that all Magento projects use by default to connect to the integration database. This means that developers won't need to set this locally, resulting on a smoother experience.

Notes: There is no hurry for this to be released at the moment, I am planning on announcing this when I am back from paternity (mid december) in a dev meeting demo. I also need to install mysqldump tool to the docker php image but that's a different repo.

### CODE REVIEW

File: wf2/src/tests/up_cmd.rs [hunk 1]
```diff
@@ -20,7 +20,7 @@ mod tests {
 
     fn test_up(args: Vec<&str>, expected: &str) {
         let cli_output = CLIOutput::from_input(CLIInput::_from_args(args));
-        match cli_output.unwrap().tasks.unwrap().get(10).unwrap() {
+        match cli_output.unwrap().tasks.unwrap().get(11).unwrap() {
```
Comment (mikeymike) [inline wf2/src/tests/up_cmd.rs]: what's the significance of `11` here ?

Comment (dcabrejas) [reply wf2/src/tests/up_cmd.rs]: I added a new task so the counter of tasks went up to 11.

Comment (mikeymike) [reply wf2/src/tests/up_cmd.rs]: ah it's a test, odd check to do but okay 😂

**Comment by dcabrejas** [comment]
I'm not sure why all the tests are failing 😂

**Pull request merged**